src/main/java/net/apnic/rdap/resource/ResourceConfiguration.java

```diff
@@ -0,0 +1,57 @@
+package net.apnic.rdap.resource;
+
+import net.apnic.rdap.autnum.AutnumStatsResourceLocator;
+import net.apnic.rdap.autnum.AsnRange;
+import net.apnic.rdap.domain.Domain;
+import net.apnic.rdap.domain.DomainStatsResourceLocator;
+import net.apnic.rdap.entity.Entity;
+import net.apnic.rdap.ip.IPStatsResourceLocator;
+import net.apnic.rdap.nameserver.NameServer;
+
+import net.ripe.ipresource.IpRange;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ResourceConfiguration
+{
+    private ResourceLocator<AsnRange> autnumResourceLocator =
+        new AutnumStatsResourceLocator();
+
+    private ResourceLocator<IpRange> ipResourceLocator =
+        new IPStatsResourceLocator();
+
+    private ResourceLocator<Domain> domainResourceLocator =
+        new DomainStatsResourceLocator(ipResourceLocator);
+
+    @Bean
+    public ResourceLocator<AsnRange> autnumResourcelocator()
+    {
+        return autnumResourceLocator;
+    }
+
+    @Bean
+    public ResourceLocator<Domain> domainResourceLocator()
```

Comment:
{suggestion} If you put the actual object creation in here then you'll need to @Autowire this method, passing the `ipResourceLocator`.